advanced_bank_reconciliation/advanced_bank_reconciliation/doctype/advance_bank_reconciliation_tool/advance_bank_reconciliation_tool.py

```
@@ -1301,6 +1301,227 @@ def create_payment_entries_for_invoices(bank_transaction_name, invoices, auto_re
+					)
+					
+					batch_vouchers.append({
+						"payment_doctype": "Payment Entry",
+						"payment_name": payment_entry.name,
+						"amount": allocated_amount,
+					})
+					
+					processed += 1
+					
+				except Exception as e:
+					logger.error(f"Error processing invoice {invoice_data.get('name')}: {str(e)}", exc_info=True)
```

Ruff (0.14.1)

Use explicit conversion flag

Replace with conversion flag

(RUF010)